custom/icds/tasks/sms.py

```diff
@@ -74,8 +74,9 @@ def send_custom_sms_report(start_date, end_date, email):
     recipients = [email]
     filename = call_command('get_icds_sms_usage', 'icds-cas', str(start_date), str(end_date))
     report_tracker = CustomSMSReportTracker()
+    report_tracker.add_report_to_list(start_date, end_date)
+    filename = call_command('get_icds_sms_usage', 'icds-cas', start_date, end_date)
```

Comment:
call_command should be in try block, I think that is incorrect. Possibly explains why Akshita received no email. The method should catch any possible exception and report to the user, but that wont happen here,

Reply:
Agreed. That was an oversight in the previous task.
I will fix it.
Thanks for catching that.